src/engine/renderer/gl_shader.h

```diff
@@ -30,7 +30,7 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 #define USE_UNIFORM_FIREWALL 1
 
 // *INDENT-OFF*
-static const unsigned int MAX_SHADER_MACROS = 9;
+static const unsigned int MAX_SHADER_MACROS = 10;
 static const unsigned int GL_SHADER_VERSION = 4;
```

Comment:
It seems you need to increase GL_SHADER_VERSION as well (which means the version of the shader cache files), because changing MAX_SHADER_MACROS changes the header struct's layout. When I run 0.54.1 after running this branch, my Nvidia driver crashes.

Reply:
Ah! Good to know! Maybe we should add `MAX_SHADER_MACROS` to the checksum?

Reply:
Not the checksum exactly, but yes it would be good to add it to the header struct.

Reply:
Yes, “_ in a way it contributes to the data used to compute the checksum_” or something like that.

Reply:
The GLSL source code is checksummed. Other data are stored directly into the header and simply compared for equality.

Reply:
I see.